CHANGELOG.md

```diff
@@ -25,11 +25,9 @@
 
 ### App Center Data
 
- The App Center Data service provides functionality enabling developers to persist app data in the cloud in both online and offline scenarios. This enables you to store and manage both user-specific data as well as data shared between users and across platforms.
+Version 2.4.0 of the App Center React Native SDK includes a new module: Data.
```

Comment:
Doesn't seem consistent with previous messages.

Reply:
Do we need this line? You don't need to mention `Version 2.4.0` in 2.4.0 change log. 🤔